### ISSUE TITLE
[Bug]: App crashed

### ISSUE THREAD

**Issue by SoytanM** [AndroidIDEOfficial/AndroidIDE#1127]
### What happened?

The app is crashing when I am using overwrite super class method options.
### What's the expected behavior?

Overwrite super class method

### What version of AndroidIDE you're using?

v2.5.1 (debug builds)

### Relevant log output

```shell
AndroidIDE crash report
Manufacturer: Xiaomi
Device: RedmiNote7Pro
ABI: arm64-v8a
SDK version: 29
App version: v2.5.1-beta (251)

 Stacktrace: 
java.lang.StringIndexOutOfBoundsException: Column 8 out of bounds. line: 133 , column count (line separator included):3
	at io.github.rosemoe.sora.text.Content.checkLineAndColumn(Unknown Source:44)
	at io.github.rosemoe.sora.text.Content.insertInternal(Unknown Source:6)
	at io.github.rosemoe.sora.text.Content.insert(Unknown Source:9)
	at com.itsaky.androidide.lsp.java.actions.generators.OverrideSuperclassMethodsAction$$ExternalSyntheticLambda0.run(Unknown Source:103)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7592)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
Hey there @SoytanM,

Can you tell me what method or methods are you trying to override? It would be helpful for me to reproduce this issue.

**Comment by SoytanM** [comment]
> Hey there @SoytanM,
> 
> Can you tell me what method or methods are you trying to override? It would be helpful for me to reproduce this issue.

**Comment by SoytanM** [comment]
AsyncTask

**Comment by SoytanM** [comment]
[Watch this video](https://drive.google.com/file/d/1vTS2ILMJz9G8kU3p-8-XqknR6N1MoviS/view?usp=drivesdk)

**Comment by itsaky** [comment]
Thanks for the video. That was helpful.

**Comment by itsaky** [comment]
This has been fixed in 1e07da7.
